README.md

```diff
@@ -68,6 +69,27 @@ The webapp/webpage is atm just to show the QRcode with out of band invitation fo
 
 **Open the webpage for develop** - open> `file:///.../webapp/index-fastopt.html`
 
+### Configure the Mediator
```

Comment:
It would be nice to give examples of the correct values for all configuration variables in one codeblock.

Reply:
We already give all the defaults here `The default configuration is set up [application.conf](/mediator/src/main/resources/application.conf).`

Although I was thinking that maybe we should not give them the private keys for the DID because almost everyone will use it...